exasol/analytics/audit/audit.py

```diff
@@ -225,5 +225,5 @@ def create_query(self) -> ModifyQuery:
             query_string=self.create_statement,
             db_object_type=DbObjectType.TABLE,
             db_object_name=self.name,
-            db_operation_type=DbOperationType.INSERT,
+            db_operation_type=DbOperationType.CREATE_IF_NOT_EXISTS,
```

Comment:
if the audit table doesn't exist and you try to log the creation this will fail. We need to set audit ot false.

Reply:
This is probably a misunderstanding. 
The create-statement for the Audit Table itself is never meant to be logged to the Audit Table.

The implementation of the `AuditQueryHandler` therefore does not (and should never) pass this query to `AudiTable.augment()`. Additionally, attribute `audit` to the constructor is set to the default value `False`.